include/wtmpdb.h

```diff
@@ -41,6 +41,8 @@
 #define NSEC_PER_USEC ((uint64_t) 1000ULL)
 #define USEC_PER_SEC  ((uint64_t) 1000000ULL)
 
+#define LOGROTATE_DAYS 20
```

Comment:
Please make this internal to src/wtmpdb.c, since this header file is for external interfaces, but the define is only useful internal.